packages/config/src/android/AllowBackup.ts

```diff
@@ -0,0 +1,37 @@
+import { ExpoConfig } from '../Config.types';
+import { Document } from './Manifest';
+
+export function getAllowBackup(config: Pick<ExpoConfig, 'android'>): boolean {
+  // Defaults to true.
+  // https://docs.expo.io/versions/latest/config/app/#allowbackup
```

Comment:
@cruzach - can you confirm that this is intentional?

Reply:
yep, bc we want to match Android's default for `android:allowBackup`- https://developer.android.com/guide/topics/manifest/application-element